tools/constants.py

```diff
@@ -45,6 +45,9 @@
 SUBDIVISION_NAME = "subdivision_name"
 MUNICIPALITY = "municipality"
 BOUNDING_BOX = "bounding_box"
+STATUS = "status"
+ACTIVE = "active"
+FEATURES = "features"
```

Comment:
I would add all feature constants here so users can use them in their scripts.
```suggestion
FEATURES = "features"
FARES_V2 = "fares-v2"
FARES_V1 = "fares-v1"
FLEX_V2 = "flex-v2"
FLEX_V1 = "flex-v1"
PATHWAYS = "pathways"
OCCUPANCY = "occupancy"
```

Reply:
added